android/filament-android/src/main/include/JavaScriptCore/APICallbackFunction.h

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (C) 2013, 2016 Apple Inc. All rights reserved.
```

Comment:
Here I pulled in all the sources.
EXGL uses a script to pull in just the necessary header files, and on a specific version of the library.
https://github.com/expo/expo/blob/master/packages/expo-gl-cpp/android/build.gradle#L79-L80